core/src/main/java/com/kin/ecosystem/core/network/model/Body.java

```diff
@@ -0,0 +1,91 @@
+package com.kin.ecosystem.core.network.model;
+
+/*
+ * Kin Ecosystem
+ * Apis for client to server interaction
+ *
+ * OpenAPI spec version: 1.0.0
+ *
+ *
+ * NOTE: This class is auto generated by the swagger code generator program.
+ * https://github.com/swagger-api/swagger-codegen.git
+ * Do not edit the class manually.
+ */
+
+
+import com.google.gson.annotations.SerializedName;
+import java.util.Objects;
+
+/**
+ * Body
+ */
+public class Body {
```

Comment:
what's the purpose of this `Body` class? it seems like a wrapper for error, we can't we use Error directly?

Reply:
We can use, but for future maybe we will want to add something extra without affecting the Error class. Also this is something generated from swagger and i don't wanna change that